pkg/fileutil/fileutil_test.go

```diff
@@ -166,3 +168,45 @@ func TestDirPermission(t *testing.T) {
 		t.Errorf("expected error, got nil")
 	}
 }
+
+func TestRemoveMatchFile(t *testing.T) {
+	tmpdir := t.TempDir()
+	defer os.RemoveAll(tmpdir)
+	f, err := ioutil.TempFile(tmpdir, "tmp")
```

Comment:
`io/ioutil ` has already been deprecated in golang 1.16, please use `os.CreateTemp` instead.

FYI. https://go.dev/doc/go1.16#ioutil

Reply:
addressed.